### ISSUE TITLE
Validation percentage specification when creating a dataset through text files

### ISSUE THREAD

**Issue by GiuliaP** [NVIDIA/DIGITS#280]
Hi,

I specified my dataset through train and test text files. What I do not understand is how the validation set is handled in this situation: when specifying the dataset with folder paths, it is possible either to specify the percentage for the validation set or the folder with validation images. When specifying the dataset through text files, it seems possible only to specify the validation set through a text file. Is it possible to specify only a percentage also in this case?
If I do not provide a text file for validation, what is the validation set?

At present I am specifying only a train and a text file as I mentioned: when I create my model and link it to this dataset however, I do not understand why, even if in the data layer I put both the train and the test phases, in the resulting train_val.prototxt I only see the train source db, and I am not able to see the accuracy evolution during training, but just the learning rate plot.

Thank you
Giulia

**Comment by lukeyeager** [comment]
The first half of your question seems to be a duplicate of #107.

I'm confused about what you're trying to do in the second half of your question. If you don't have a validation set of images, there's no point in having a `TEST` phase (read: validation phase) during training. Are you saying you want to train on a set of images, and then calculate the accuracy on that same set of images?

**Comment by GiuliaP** [comment]
You're right, it is a duplicate, I'm sorry I didn't notice.

Regarding the second part, I was confused (I thought that a validation set would have been used during the training even without specifying it in the dataset) but now I understand: using text files, without a validation file there is no validation set, the TEST phase is ignored accordingly in the train_val.prototxt, and therefore also the accuracy plot is missing. Thank you!!
